resources/k8s-artifacts/choreo-connect/config-toml-configmap-for-eventhub.yaml

```diff
@@ -142,6 +142,10 @@ data:
         username = "admin"
         password = "$env{enforcer_admin_pwd}"
 
+    [enforcer.authHeader]
```

Comment:
Shouldn't this be come under [security.enforcer]?

Reply:
I suppose [enforcer.authHeader] is okay. When looking at the existing config.toml it has this config format (e.g. [enforcer.jwtIssuer], [enforcer.jwtGenerator]).

Reply:
Mmmm. Since we have added security related configs under  [security]. Doesn't it good to move at there?

Reply:
+1 done